postgres/src/main/scala/zio/sql/postgresql/PostgresModule.scala

```diff
@@ -49,6 +49,8 @@ trait PostgresModule extends Jdbc { self =>
     val Now                         = FunctionDef[Any, ZonedDateTime](FunctionName("now"))
     val StatementTimestamp          = FunctionDef[Any, ZonedDateTime](FunctionName("statement_timestamp"))
     val TransactionTimestamp        = FunctionDef[Any, ZonedDateTime](FunctionName("transaction_timestamp"))
+    val Encode                      = FunctionDef[(String, String), String](FunctionName("encode"))
+    val Decode                      = FunctionDef[(String, String), String](FunctionName("decode"))
```

Comment:
Encode and decode in PostgreSQL operate on binary string type (`bytea`). The documentation describes the difference https://www.postgresql.org/docs/13/datatype-binary.html. I'm wondering if it wouldn't be more appropriate to represent these as `Chunk[Byte]` and introduce tests that include non-printable octets.

Reply:
I'm definitely with you about it and I had the same doubt but I eventually, opted to use `String` since also `md5`, that is supposed to ingest a `bytea`, was defined as:
```
val Md5                         = FunctionDef[String, String](FunctionName("md5"))
```
probably we should also change this one then. WDYT?

I'll try to adapt them all to `Chunk[Byte]`.

Reply:
Right, apparently I missed that. So yeah, once we have `encode`/`decode` working we can fix others.

Reply:
I pushed my proposal. I had to make invariant `zio.sql.TypeTagModule.TypeTag` and other two as a consequence in order to avoid this compilation error when mapping over the chunk:
```
.../zio-sql/postgres/src/main/scala/zio/sql/postgresql/PostgresModule.scala:120:64
value map is not a member of Any
        case tt @ TByteArray      => render(tt.cast(lit.value).map("""\%03o""" format _).mkString("E\'", "", "\'"))
```

Please suggest alternatives if you think this way is not right.